_posts/2016-12-23-gitbucket-4.8.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "GitBucket 4.8 released!"
+date: 2016-12-23 00:00:00
+image: /images/gitbucket-4.8/repository_search_box.png
+categories: gitbucket
+---
+
+Today, we are pleased to announce the availability of new version of Scala based Git platform [GitBucket 4.8](https://github.com/gitbucket/gitbucket/releases/tag/4.8)!
+
+## Search for repository names
+
+The search box on the global header now works for search repository:
+
+![Repository search box on the header]({{site.baseurl}}/images/gitbucket-4.8/repository_search_box.png)
+
+Also the filter box on the dashboard sidebar to filter repositories has been available:
```

Comment:
would have formulated `Also a filter box on the dashboard sidebar to filter repositories have been introduced:`